playground/src/modal-test.ts

```diff
@@ -0,0 +1,6 @@
+import './main.css'
+import '../../src/main'
+import { createApp } from 'vue'
+import App from './modal-test.vue'
+
+createApp(App).mount('#app')
```

Comment:
is there a way we could dynamically generate this file and the `.html` file? we should probably move tests to their individual pages and off of `test.html`

Reply:
Yeah we should probably make the testbed into a single page app to avoid all this boilerplate. We could pretty easily do it with Nuxt or Sveltekit in the near future.

Reply:
should i make a ticket for this?

Reply:
Yeah that would be great! If you do add a comment here linking the ticket you created.

Reply:
https://viam.atlassian.net/browse/APP-1383